go/worker/storage/committee/node.go

```diff
@@ -891,6 +881,18 @@ func (n *Node) worker() { // nolint: gocyclo
 	}
 	close(n.initCh)
 
+	// We are now ready to service requests.
```

Comment:
It would be cool if this could wait until the storage is actually synced close enough with the current round (e.g., there could be no checkpoints or the node would still need to sync quite a few diffs from the last checkpoint).